src/tagreader.cpp

```diff
@@ -457,7 +457,6 @@ class TagFileParser
         case InMember:
         case InPackage:
         case InDir:
-          if (m_curString.endsWith("autotoc_md")) return;
           break;
```

Comment:
Removing this line will give problems when a tag file is used from the 1.8.16 version of doxygen is used as here in the `p1.tag` file of the given example there is the line:
```
<docanchor file="md_intro" title="Description">autotoc_md0</docanchor>
```

Reply:
I dont really understand this. As far as I can see it starts with autotoc_md, not ends with. It ends with 0 but thats not what we checked for.

Reply:
I think you are right, probably a bug no one noticed, the tag file indeed gives quite clearly `autotoc_md0` so it should be `startsWith` instead of `endsWith`.

Reply:
Yeah and because it went unnoticed for so long it thought it's unnessasary and removed it. If you want I can add some code and check if we are in the doxygen Mode and than check if it starts with `autotoc_md`.

Reply:
I think it is better to add the correct test.

Reply:
I'll do it today tonight(about when I commited yesterday)